arangod/Replication2/ReplicatedLog/Supervision.cpp

```diff
@@ -388,7 +388,13 @@ auto checkReplicatedLog(LogTarget const& target,
                         ParticipantsHealth const& health) -> Action {
   if (!maybePlan) {
     // The log is not planned right now, so we create it
-    return AddLogToPlanAction(target.id, target.participants);
+    LogPlanTermSpecification term;
+    term.config = target.config;
+    term.term = LogTerm{1};
+    auto leader = target.participants.begin()->first;
+    auto rebootId = health._health.at(leader).rebootId;
```

Comment:
You have to check whether the log has been created with any participants for this to not fail rather ungracefully; we could of course just `ErrorAction` if one tries creating a log without any participants (or, not enough participants for the config) ?

Reply:
I thought I could remove `CreateIntialTermAction`, but while doing that the above showed up; we can probably still remove `CreateInitialTermAction` because we can just create the log with `LogTerm{1}` regardless

Reply:
Yes, in the end I don't care, how its done. Important to me is, that we safe round trips when the result of those round trips is already known.

We can close this PR. Just copy the part from Agency/Supervision to make writes of ReplicatedState supervision visible to ReplicatedLog supervision.